help/destinations/catalog/email-marketing/sendgrid.md

```diff
@@ -0,0 +1,201 @@
+
+## Connect to destination {#connect}
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). Steps specific to this destination are illustrated in detail below.
+
+* Within the [!DNL Adobe Experience Platform] console, navigate to *Destinations*.
+
+* Select the *Catalog* tab and search for *Sendgrid*. Then select *Activate Segments*.
+
+![](../../assets/catalog/email-marketing/sendgrid/02-catalog.jpg)
+
+* You will be shown a wizard where the first screen is to choose a destination. For setting up this connector, let us create a new destination by selecting *Configure new destination*.
+
+![](../../assets/catalog/email-marketing/sendgrid/03.jpg)
+
+* Select the *New Account* option and fill in the *Bearer Token* value which is the Sendgrid *API Key* value we had previously copied within [prerequisites](#prerequisites "Goto prerequisites").
+
+![](../../assets/catalog/email-marketing/sendgrid/04.jpg)
+
+* Upon selecting *Connect to destination*, you will be shown the next [step](#connection-parameters-parameters) to fill in additional information fields.
+
+### Connection parameters {#parameters}
+
+While [setting up](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/connect-destination.html?lang=en) this destination, you must provide the following information:
+
+*  **[!UICONTROL Name]**: The name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: An optional description that will help you identify this destination in the future.
+
+![](../../assets/catalog/email-marketing/sendgrid/05.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/06.jpg)
+
+## Activate segments to this destination {#activate}
+
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination. Steps specific to this destination are illustrated in detail below.
+
+![](../../assets/catalog/email-marketing/sendgrid/13.png)
+
+![](../../assets/catalog/email-marketing/sendgrid/14.jpg)
+
+> [!IMPORTANT]
+> The *Source* field attribute *"Email"* should be mapped to the *Target* field attribute "external_id" as shown in the subsequent step.
+
+![](../../assets/catalog/email-marketing/sendgrid/15.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/16.jpg)
+
+* Similarly we map the remaining AEP fields and attributes to the Sendgrid attributes. For e.g. we are mapping the person object's first name atrribute below
+
+![](../../assets/catalog/email-marketing/sendgrid/17.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/18.jpg)
+
+* A preview of some of the mapping's is as below
+
+![](../../assets/catalog/email-marketing/sendgrid/19.jpg)
+
+* The comprehensive list of the mappings that would be required to be setup is listed below.
```

Comment:
Why are the mappings "required" to be set up? Must all these mappings be set up? Maybe a better way of phrasing it would be "The comprehensive list of the mappings that can be set up to be exported to Sendgrid is listed below."

Reply:
Yes most are non optional. Have used your verbiage.